src/DTO/TrickDTO.php

```diff
@@ -18,4 +18,7 @@ class TrickDTO
 
     #[Assert\Url]
     public $imageUrl;
+
+    #[Assert\Url]
+    public $videoUrl;
```

Comment:
Maybe you should use array of dto for image and video.
```suggestion
    public $images;
    public $videos;
```